src/components/Loader/PulseLoader.jsx

```diff
@@ -0,0 +1,3 @@
+export default function PulseLoader() {
```

Comment:
Ah I kept seeing you import and use this component throughout but wasn't sure what it was. Makes a lot more sense now 😂

Reply:
One of my stretch goals is to give all the elements a fancy loading widget once (and if) I finish everything else up. That component there is just to remind me of that.

Reply:
Haha with the name you'd given that I figured you would have some sort of fancy effect going on during the load time and such but couldn't see where you'd coded it. Yeah makes sense, will be quite cool if you get around to implementing that!